chance.js

```diff
@@ -2294,6 +2294,28 @@ options,
         return this.pick(this.currency_types());
     };
 
+    // return random world amount of currency
+    Chance.prototype.amount = function (options) {
```

Comment:
Please add documentation to the `/docs/finance` section

Reply:
added `/docs/finance/amount.md`